utils/cq/requirements.txt

```diff
@@ -4,9 +4,9 @@ pyenchant
 ## https://github.com/pycqa/flake8/issues/1389 https://github.com/PyCQA/flake8/pull/1720
 flake8==7.1.1
 isort==5.13.2
-pylint==3.3.2
+pylint==3.3.3
 yamllint==1.35.1
-codespell==2.3.0
+codespell==2.4.0
```

Comment:
@grom72 This is why the codespell warnings crept in. Even though our GHA installs this requirements.txt, the workflow we use installs its own requirements, which pulls the latest codespell:
https://github.com/codespell-project/actions-codespell/blob/406322ec52dd7b488e48c1c4b82e2a8b3a1bf630/Dockerfile#L10

And I don't see an obvious way to configure that.

Reply:
Would implementing `codespell>=2.4.0` - similar to https://github.com/codespell-project/actions-codespell/blob/406322ec52dd7b488e48c1c4b82e2a8b3a1bf630/requirements.txt#L1 - solve this and also avoid a bunch of these tickets?

Reply:
It might cut down on these PRs (I'm not sure if dependabot would still try to update when a newer version comes out).

But it doesn't solve the problem of new warnings.
For example:
1. Someone has codespell 2.4.0, which is valid because `codespell>=2.4.0`
2. They commit and push - no issues in the githooks
3. The GHA runs with codespell 2.5.0, which is valid because `codespell>=2.4.0`
4. The GHA fails because there are new warnings

To prevent this, the action we are using needs to support pinning the codespell version in some way

Reply:
The alternative solution could be to regularly (daily) run all PR-only workflows on the `master' branch to detect such situations before someone gets hit. 
We have recently a similar problem in the PMDK project when the default clang version changed producing a set of new warnings: https://github.com/pmem/pmdk/issues/6134

Reply:
I don't think that would help here. The problem is that the GHA implicitly uses the newest version. As soon as the new version is available, PRs will use it and potentially fail.